fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductAnalysis.kt

```diff
@@ -347,55 +373,93 @@ private enum class Highlight(
     ),
 }
 
+private class ProductAnalysisPreviewModel(
+    val productRecommendationsEnabled: Boolean,
```

Comment:
To be in sync with #3618 could we make this nullable.

Reply:
kk, I'll incorporate those changes once it lands 👍